modules/learn/pages/services-and-indexes/services/services.adoc

```diff
@@ -1,99 +1,130 @@
+Executes code in response to document mutations or on scheduled timers.
+* xref:learn:services-and-indexes:services/backup-service.adoc[Backup Service]: Schedules full and incremental backups for specific buckets or all buckets in a cluster.
+Supports merging and pruning of existing backups.
+
+With multi-dimensional scaling, you can deploy, maintain, and provision these services independently to optimize performance as business needs and workloads change.
 
 [#setting-up-services]
 == Setting Up Services
 
-Services are set up on a per-node basis.
-Each node can run at most one instance of a service.
-Each node can run any number of services, up to the maximum, which is seven.
-In Couchbase Enterprise Server Version 7.6+, a node can be configured to run _no_ service.
-The _Data Service_ must run on at least one node of the cluster.
-Some services are interdependent and therefore require at least one instance of
-each of their dependencies to be running on the cluster — (for example, the
-_Query Service_ depends on the _Index Service_ and on the _Data Service_).
+Services are configured per node.
+Each node can run one instance of a service and up to seven services in total.
+In Couchbase Enterprise Server 7.6 and later, you can configure a node to run no services.
+
+The Data Service must run on at least one node in the cluster.
+Some services are interdependent and require at least one instance of each dependency to be running.
+For example, the Query Service depends on the Index and Data services.
+
+When you initialize the first node in a cluster, that node's service configuration becomes the default for nodes added later.
+You can change the default configuration for individual nodes by removing services, adding services, or both.
+
+In Couchbase Server 8.0 and later versions, you can add or remove the following non-Data Services on-demand on existing nodes in a cluster, without adding or removing nodes.
+
+* index (Index Service).
+* n1ql (Query Service).
+* fts (Search Service).
+* cbas (Analytics Service).
+* eventing (Eventing Service).
+* backup (Backup Service).
+
+Then a rebalance operation is automatically triggered to evenly distribute the service workload across the nodes.
+For information about adding or removing non-Data Services on an existing node, see xref:manage:manage-nodes/modify-services-and-rebalance.adoc[Modify Services and Rebalance].
+
+Adding or removing of kv (Data Service) on an existing node is supported only through adding or removing nodes.
+
+To add a Data Service on a node from the UI:
+
+* xref:manage:manage-nodes/add-node-and-rebalance.adoc[Add a New Node and Rebalance] along with enabling the Data Service.
+
+* xref:manage:manage-nodes/remove-node-and-rebalance.adoc[Remove a Node and Rebalance].
+Then xref:manage:manage-nodes/add-node-and-rebalance.html[add the node] back to the cluster along with enabling the Data Service.
```

Comment:
This is a comment for lines 68 through 71.

If I want to add the Data service on an existing node that does not currently have the Data service, all I need to do is lines 70 and 71 -- remove the node, then add the node back to the cluster along with enabling the Data Service.

So, if you are trying to say that there are 2 ways to do this -- add a new node with a data service (line 68)   OR remove a node (that does not have a data service), then, add it back in along with enabling the data service (lines 70 and 71), you need an OR between the two.

Either fix the confusion by removing line 68 or fix the confusion by adding an OR on line 69.